packages/server/assets/core/typedefs/admin.graphql

```diff
@@ -0,0 +1,34 @@
+type AdminUserList {
+  items: [BaseUser!]!
+  cursor: String
+  totalCount: Int!
+}
+
+type ServerStatistics {
+  totalProjectCount: Int!
+  totalUserCount: Int!
+  totalPendingInvites: Int!
+}
+
+type AdminQueries {
+  userList(
+    limit: Int! = 25
+    cursor: String = null
+    query: String = null
+    role: ServerRole = null
+  ): AdminUserList! @hasScope(scope: "users:read")
+
+  projectList(
+    query: String
+    orderBy: String
+    visibility: String
+    limit: Int! = 25
+    cursor: String = null
+  ): ProjectCollection!
+
+  serverStatistics: ServerStatistics! @hasScope(scope: "server:stats")
```

Comment:
If the only reason for this new field is so that it's organized together with the rest of the new queries, maybe we can just re-use the existing `ServerStats` return type so that all of the pre-existing resolvers can be re-used?